### DISCUSSION TITLE
numbering issue

### DISCUSSION THREAD

**Discussion by dreckguy** [QualiTorque/qualitorque.github.io#51] [Documentation requests] [answered]
In the [Blueprint yaml page](https://docs.qtorque.io/blueprint-designer-guide/blueprints) There is a listing of 
the ways "sources can be defined" but there are 2 items with the number 1 instead of 1 and 2.

![image](https://user-images.githubusercontent.com/15980885/175813113-bd553d54-ff80-4ac9-b619-8b0d304dafa7.png)

**Comment by Shnitselon** [comment, marked as answer]
Hi Ophir,

Thanks a lot for the feedback.
Fixed
